### PR TITLE
Update information-display.js

### PR DESCRIPTION
**Pull request by sebagabs**
*`script-two.js`

- Add two new functions that handle the display of information
- Add function calls so they run with info.html

### CODE REVIEW

**Review by SamRod33 — COMMENTED** [review]
Hey, some of these same committed files appeared in another PR. Try to avoid this, and if you can, undo these commits. Please make sure before moving onto different branches, add, commit, and push the files on the current branch. Also, when making a new branch, if it is not dependent on the current branch, switch into main, then make the new branch

**Review by SamRod33 — APPROVED** [review]

**Comment by sebagabs** [comment]
> Hey, some of these same committed files appeared in another PR. Try to avoid this, and if you can, undo these commits. Please make sure before moving onto different branches, add, commit, and push the files on the current branch. Also, when making a new branch, if it is not dependent on the current branch, switch into main, then make the new branch

I tried doing that, since I've had this issue before, but for some reason I did it wrong. Thanks!

**Comment by GoogleyGabriel** [comment]
> > Hey, some of these same committed files appeared in another PR. Try to avoid this, and if you can, undo these commits. Please make sure before moving onto different branches, add, commit, and push the files on the current branch. Also, when making a new branch, if it is not dependent on the current branch, switch into main, then make the new branch
> 
> I tried doing that, since I've had this issue before, but for some reason I did it wrong. Thanks!

Sam is partially right, but the issue is:

1. Make sure that your branch is updated. In this branch, pull from the other branch. `git pull origin UpdateFrontEnd`
2. The reason that it is showing this file is because it is comparing this PR to main, not to UpdateFrontEnd. On this branch, you do not want to merge to main, you want to merge to the branch you branched off of (UpdateFrontEnd). When UpdateFrontEnd gets merged, this branch will automatically update so that it now merges into main (since the other branch has been merged). To fix this issue, you can press on edit (at the top of this page)

Click on edit:
image

and then change `base: main` to the branch it is based off if, in this case, UpdateFrontEnd
image

**Comment by sebagabs** [comment]
> > > Hey, some of these same committed files appeared in another PR. Try to avoid this, and if you can, undo these commits. Please make sure before moving onto different branches, add, commit, and push the files on the current branch. Also, when making a new branch, if it is not dependent on the current branch, switch into main, then make the new branch
> > 
> > 
> > I tried doing that, since I've had this issue before, but for some reason I did it wrong. Thanks!
> 
> Sam is partially right, but the issue is:
> 
> 1. Make sure that your branch is updated. In this branch, pull from the other branch. `git pull origin UpdateFrontEnd`
> 2. The reason that it is showing this file is because it is comparing this PR to main, not to UpdateFrontEnd. On this branch, you do not want to merge to main, you want to merge to the branch you branched off of (UpdateFrontEnd). When UpdateFrontEnd gets merged, this branch will automatically update so that it now merges into main (since the other branch has been merged). To fix this issue, you can press on edit (at the top of this page)
> 
> Click on edit:
> image
> 
> and then change `base: main` to the branch it is based off if, in this case, UpdateFrontEnd
> image

Just did that; it worked. Thanks!

File: project/chrome-extension/script-two.js
Comment (GoogleyGabriel) [inline project/chrome-extension/script-two.js:28]: Do not leave commented out code when merging.

File: project/chrome-extension/script-two.js
Comment (GoogleyGabriel) [inline project/chrome-extension/script-two.js:35]: Same as above

**Review by GoogleyGabriel — CHANGES_REQUESTED** [review]
Also, pick descriptive filenames. script-two.js does not tell me much about what this script does. Furthermore, why do you need a new .js file? Why can't you just add this to script.js?

File: project/chrome-extension/script-two.js
Comment (sebagabs) [reply project/chrome-extension/script-two.js:28]: We left these comments on purpose so that @SamRod33 and @SofiaPSU could know where the information from the JSON should go. After those were implemented they would be removed.

File: project/chrome-extension/script-two.js
Comment (sebagabs) [reply project/chrome-extension/script-two.js:35]: We left these comments on purpose so that @SamRod33 and @SofiaPSU could know where the information from the JSON should go. After those were implemented they would be removed.

File: project/chrome-extension/script-two.js
Comment (GoogleyGabriel) [reply project/chrome-extension/script-two.js:28]: In that case, this comment should be a todo:

`// TODO(username_for_who_has_to_fix_this): Add to the description by using description.innerText`

Also, do not forget that between comments and the first word, there must be a space.

**Comment by GoogleyGabriel** [comment]
I noticed that Claudia created the file, so ignore the comment about the filename. I commented it in Claudia's PR instead.

File: project/chrome-extension/script-two.js
Comment (sebagabs) [reply project/chrome-extension/script-two.js:28]: Noted, thank you!

**Comment by sebagabs** [comment]
> Also, pick descriptive filenames. script-two.js does not tell me much about what this script does. Furthermore, why do you need a new .js file? Why can't you just add this to script.js?

We created another JavaScript file because we need those specific functions in `script-two.js` to run when `info.html` is loaded. Having only one script was giving us some trouble figuring out this so we decided to just have two separate ones.

I made the requested changes on CloudShell but when I attempt to push to this branch, this happnes:
image

Any idea why @GoogleyGabriel?

**Comment by GoogleyGabriel** [comment]
It was an issue on my part, please try to push again.

**Comment by sebagabs** [comment]
@GoogleyGabriel just pushed the requested changes. Let me know of anything else.

**Review by GoogleyGabriel — APPROVED** [review]

**Comment by GoogleyGabriel** [comment]
Remember, do not merge until the base branch has been merged.

**Comment by sebagabs** [comment]
Yes, thank you!

**Comment by GoogleyGabriel** [comment]
Also, update the title since the file is named differently now

**Comment by sebagabs** [comment]
> Also, update the title since the file is named differently now

Just did so. BTW: the UpdateFrontEnd branch still indicates that changes have been request even tho I have completed them. So it will not let me merge them until you indicate otherwise.

**Comment by GoogleyGabriel** [comment]
> > Also, update the title since the file is named differently now
> 
> Just did so. BTW: the UpdateFrontEnd branch still indicates that changes have been request even tho I have completed them. So it will not let me merge them until you indicate otherwise.

Approved it -- you should be able to merge now

**Pull request merged**
